app/lib/features/tasks/actions/create_task.dart

```diff
@@ -0,0 +1,433 @@
+                });
+              },
+              icon: const Icon(Icons.close),
+            ),
+          ],
+        ),
+        const SizedBox(height: 5),
+        TextFormField(
+          readOnly: true,
+          key: CreateTaskWidget.dueDateField,
+          decoration: InputDecoration(
+            hintText: lang.dueDate,
+            suffixIcon: IconButton(
+              onPressed: selectDueDate,
+              icon: const Icon(Icons.calendar_month),
+            ),
+          ),
+          onTap: selectDueDate,
+          controller: _taskDueDateController,
+        ),
+        Row(
+          mainAxisAlignment: MainAxisAlignment.end,
+          children: [
+            ActerInlineTextButton(
+              key: CreateTaskWidget.dueDateTodayBtn,
+              onPressed: () => setState(() {
+                final date = DateTime.now();
+                selectedDate = date;
+                _taskDueDateController.text = taskDueDateFormat(date);
+              }),
+              child: Text(lang.today),
+            ),
+            ActerInlineTextButton(
+              key: CreateTaskWidget.dueDateTomorrowBtn,
+              onPressed: () => setState(() {
+                final date = DateTime.now().addDays(1);
+                selectedDate = date;
+                _taskDueDateController.text = taskDueDateFormat(date);
+              }),
+              child: Text(lang.tomorrow),
+            ),
+          ],
+        ),
+      ],
+    );
+  }
+
+  Future<void> selectDueDate() async {
+    final due = await showDuePicker(
+      context: context,
+      initialDate: selectedDate,
+    );
+    if (due == null || !mounted) return;
+    final date = due.due;
+    setState(() {
+      selectedDate = date;
+      _taskDueDateController.text = taskDueDateFormat(date);
+    });
+  }
+
+  Widget _addFields() {
+    final lang = L10n.of(context);
+    final actions = [];
+    if (!showDescriptionField) {
+      actions.add(
+        ActerInlineTextButton(
+          key: CreateTaskWidget.addDescAction,
+          onPressed: () => setState(() {
+            showDescriptionField = true;
+          }),
+          child: Text(lang.description),
+        ),
+      );
+    }
+    if (!showDueDate) {
+      actions.add(
+        ActerInlineTextButton(
+          key: CreateTaskWidget.addDueDateAction,
+          onPressed: () => setState(() {
+            showDueDate = true;
+          }),
+          child: Text(lang.dueDate),
+        ),
+      );
+    }
+    if (actions.isEmpty) {
+      return const SizedBox.shrink();
+    }
+
+    return Row(
+      children: [
+        Text(lang.add),
+        const SizedBox(width: 5),
+        ...actions,
+      ],
+    );
+  }
+
+  Widget _widgetCreateButton() {
+    final lang = L10n.of(context);
+    return ElevatedButton(
+      key: CreateTaskWidget.submitBtn,
+      onPressed: addTask,
+      child: Text(lang.addTask),
+    );
+  }
+
+  Future<void> _selectTaskList() async {
```

Comment:
How about putting this in actions folder?

Reply:
We are already in the actions folder. I assume you mean make that a separate file in the actions folder!?!